src/Parser.js

```diff
@@ -1062,8 +1063,61 @@ export default class Parser {
                     body: arg,
                     star: star,
                 }, "text"), nucleus);
+        } else if (text === "$") {
+            return newDollar(nucleus);
+        }
+        // At this point, we should have a symbol, possibly with accents.
+        // If it's in the symbol table or CJK, we're done.
+        if (symbols[this.mode][text]) {
+            this.consume();
+            return newArgument(new ParseNode(symbols[this.mode][text].group,
+                text, this.mode, nucleus), nucleus);
+        } else if (cjkRegex.test(text)) {
+            this.consume();
+            return newArgument(
+                new ParseNode("textord", text, this.mode, nucleus));
+        }
+        // Decompose symbol into base and combining characters
+        // if the combined symbol is not recognized.
+        text = text.normalize('NFD');
```

Comment:
This is super cool.  I wish I would've known about this sooner.  There's no support for this in IE and Safari only got this in version 10.  We might consider using https://github.com/walling/unorm instead.

Reply:
Ah, I should have checked the [browser compatibility matrix for String.normalize](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize#Browser_compatibility).  The only problem with a polyfill is that it increases the `katex.min.js` build by 60%, from 226,836 bytes to 363,641 bytes (as any polyfill requires a huge table of Unicode symbols).

So I think this raises a serious question of whether we should:

1. Include the polyfill.
2. Not include the polyfill, and support Unicode in all supported browsers *except* IE.
3. Rewrite the PR to avoid `String.normalize`, at least in the built form.  We actually only want symbols that expand to accents within our supported set, so the symbol table we could generate at build time could be smaller than `unorm`'s.
4. Drop this PR altogether (though this is probably a desirable feature, and IE probably isn't evolving).

I think option 2 or 3 is best.  Thoughts?  I can look into option 3.

Reply:
It's in Edge so my vote would be for option 2.  If people need to support IE then they can add the polyfill themselves.  We should document this requirement.

Reply:
Sounds good.  That's the easiest!  I'll add a fallback when `normalize` isn't implemented, maybe with a warning, and add documentation.